doc/Language/packages.pod6

```diff
@@ -141,7 +141,7 @@ the setting's outermost scope, use C<UNIT::OUTER> instead.
 =head2 Interpolating into names
 X<|interpolating into names>
 
-X<|::()> You may L<interpolate|/language/packages#Interpolating> a string into a
+X<|::()> You may interpolate a string into a
```

Comment:
Wouldn't this rater be [this link](/language/packages#Interpolating_into_names)? It seems to have changed is all.

Reply:
No, the link you are suggesting is the exact section where the link in question is situated at, which means in a first paragraph of a section you link to the same section.
It in general talks about interpolating to get packages, but I think just the term "interpolating" in general should be described somewhere.

Reply:
Hum, true... But then that's where it pointed at before. So it makes sense to be removed.

Reply:
No, it makes sense to find where we describe "interpolation" as a general term and update the link to point there.

Reply:
There's no such thing right now. There's interpolation in the glossary, but that mainly refers to string. We might open another issue, which, when solved, can be pointed at.

Reply:
... and then where it would make the most sense to define it is around this same link, so no link would be necessary. Really, I think that eliminating it is OK.

Reply:
https://en.wikipedia.org/wiki/String_interpolation ?

Reply:
You mean outside the docs? OK, that might be an alternative. Again, judgement call as Richard says

Reply:
Yes, something outside the docs as we don't have a section for "interpolation" it seems, only "variable interpolation" and it does not explain the general concept.
https://github.com/Raku/doc/issues/2588 ticket (still opened due to it being too vague, alas) in a confusing, but still a way says that terms like "interpolation" should be explained somewhere and linked to, because not everyone know what does it mean and the docs intent is to serve as an educational material and/or a reference.
Hence just removing the URL will make the situation worse.